### ISSUE TITLE
Collect data and analyse the smartphone patent "wars" of 2007-2015

### ISSUE THREAD

**Issue by rgrp** [okfn/ideas#91]
tl;dr: create a comprehensive database of the litigation and results of the smartphone patent "wars". Stuff like:

* The players (which companies were involved)
* Who sued who, when (and make a timeline)
* Settlements and amounts
* Estimate of how much was spent on attorneys

## Context

From 2007 to 2015 (and ongoing) there has been a huge amount of smartphone patent litigation. This litigation is interesting for several reasons:

1. It has been large involving almost every major manufacturer and software provider and many of these companies are amongst the largest in the world and are household names (Apple, Google etc)
2. It hat has had an impact on consumers, the companies involved -- though this impact may be largely hidden from ordinary users
3. These "patent wars" are a good "existence proof" of precisely the concerns raised about software patents in terms of being a) often for trivial ideas b) resulting in holdup c) involving large transaction costs (e.g. the litigation). (cf Bessen's work on litigation costs, the work of FFII against the EU patent directive)
4. These disputes have likely influenced the growing rollback on software patents - cf the Supreme Court Alice decision - http://www.ip-watch.org/2014/06/20/us-high-court-shuts-door-on-many-software-patents/
5. Systematic information on this litigation and results is hard to find, at least in a comprehensive, consolidate format.
    * There is some great information from folks like Florian Mueller at http://www.fosspatents.com/, Ars Technica and even Endgadget e.g. http://www.engadget.com/2014/12/23/rockstar-sells-patents-ends-lawsuits/
